sdk/src/structured-reporting.ts

```
@@ -183,9 +199,11 @@ export function buildEvalRunReport(
     schemaVersion: 1,
     kind: "eval-run",
     passed,
-    ...(inputs.length > 0
-      ? { verdict: structuredEvalVerdict(inputs, passed) }
-      : {}),
+    ...(options.verdict !== undefined
+      ? { verdict: options.verdict }
+      : inputs.length > 0
+        ? { verdict: structuredEvalVerdict(inputs, passed) }
```

ESLint

[error] Delete `··`

(prettier/prettier)